### DISCUSSION TITLE
GRID大作业探测器朝向

### DISCUSSION THREAD

**Discussion by YangZiRui-THU** [physics-data/faq#126] [Q&A] [answered]
请问GRID三个探测器的朝向（入射角）是不同的还是相同的？我们能得到这个信息吗？

**Comment by YangZiRui-THU** [comment]
目前我觉得入射角是不同的，请问入射角信息我可以得到吗？

**Comment by heroxbd** [comment]
@babychen9816

**Comment by quxnb** [comment, marked as answer]
本次作业探测器与源的相对朝向已经考虑在响应矩阵中, 不需要再考虑入射角.

**Reply by YangZiRui-THU** [reply]
我的意思是大作业的额外任务“给出具有某⼀姿态(四元数)的⽴⽅星对某⼀天球⽅位Gamma源的响应矩阵”，如果不能给出具体的入射角信息，这个应该是无解的，毕竟我们也不知道探测器所有组件几何、材料，没有办法通过模拟来给出响应矩阵。

**Reply by quxnb** [reply]
入射角是 Gamma 源方位相对于探测器朝向的, 原则上你可以自己指定一个入射角, 例如均匀分布的几个入射角.

对于探测器几何等信息, 这就需要自己调研了.

**Reply by quxnb** [reply]
基本要求中的响应矩阵不考虑入射角, 直接用即可; 在提高要求中推导响应矩阵时要考虑入射角的影响

**Reply by YangZiRui-THU** [reply]
对的，我就是在说提高要求。readme中只告诉我们这个响应矩阵是用蒙特卡罗方法模拟出来的，那这个模拟用到的入射角应该可以作为已知信息吧？否则通过调研天格探测器的所有参数，并模拟计算，这个任务量已经...

**Reply by quxnb** [reply]
emmm, 我的意思是你可以给定一个入射角, 把它作为已知信息.

(如果要全部完成提高要求当然任务量巨大...但毕竟是额外给分, 视完成度而定)

**Reply by YangZiRui-THU** [reply]
好的，谢谢助教老师
